docs/multicluster/upgrade.md

```diff
@@ -37,6 +37,54 @@ for the feature in new version.
 It should have no impact during upgrade to those imported resources like Service, Endpoints
 or AntreaClusterNetworkPolicy.
 
+## Upgrade from a version prior to v1.13
+
+Prior to Antrea v1.13, the `ClusterClaim` CRD is used to define both the Cluster ID and
+ClusterSet ID for a ClusterSet. Since Antrea v1.13, the `ClusterClaim` CRD is removed, and
+the `ClusterSet` CRD solely defines a ClusterSet. The name of a ClusterSet CR must match the
+ClusterSet ID, and a new `clusterID` field specifies the Cluster ID.
+
+To upgrade from a version older than v1.13, once you apply the latest manifests for an
```

Comment:
I feel we should import new CRD version and update ClusterSet CRs first before deploying the new version MC Controller. If you plan to add auto-conversion in this release, let us update the document after that.

Reply:
Sure, I will add a new PR to do auto-conversion.

Reply:
Then how about let us add upgrade documentation with that PR and remove the changes in this PR?

Reply:
A new PR is created #5216, and update the upgrade doc in the new PR.